rusk-wallet/default.config.toml

```diff
@@ -1,5 +1,10 @@
-state = "https://nodes.dusk.network"
-prover = "https://provers.dusk.network"
+state = "https://testnet.nodes.dusk.network"
+prover = "https://testnet.provers.dusk.network"
+explorer = "https://explorer.dusk.network/transactions/transaction/?id="
```

Comment:
Change the explorer url too (both devnet and testnet)

Additionally this should section should be behind the "testnet" network, as we did for "devnet"

The default one should be for mainnet